app/javascript/tests/mocks/mockTranslations.js

```diff
@@ -259,6 +263,10 @@ const mockTranslations = {
       'congestion_or_runny_nose': {
         'name': 'Congestión nasal o un exceso de moco en la nariz',
         'notes': ''
+      },
+      'general_health': {
```

Comment:
For the `mockTranslations`, since we don't have translations for this mock symptom in these various languages, I just set the same text for all languages. If anyone believes it would be better to at least attempt to translate this symptom and include what I come up with in this file, let me know.

Reply:
I think for testing purposes this is fine

Reply:
I don't think it'd hurt to throw it in Google translate so you can make sure the correct translations are being used in testing.

Reply:
I will use Google translate for this addition then.